src/usr/local/www/system_camanager.php

```diff
@@ -250,6 +253,12 @@
 
 		$ca['descr'] = $pconfig['descr'];
 
+		if (!isset($pconfig['trusted']) || empty($pconfig['trusted'])) {
+			$ca['trusted'] = false;
+		} else {
+			$ca['trusted'] = $pconfig['trusted'];
+		}
+
```

Comment:
This should be all that it needed here: `$ca['trusted'] = isset($pconfig['trusted']);`